src/pages/MembersPage/MembersTable.tsx

```diff
@@ -79,7 +79,7 @@ type EditUserFormSchema = z.infer<typeof schema>;
 const makeCoordinapeUser = (circleId: number): QueryUser => {
   return {
     id: -1,
-    address: COORDINAPE_USER_ADDRESS,
+    profile_id: -1,
```

Comment:
Is this ok? Should this be the profile_id of the coordinape user

Reply:
This is the only blocker, im not sure if this matters or not

Reply:
yes it's defined below at line 98 ` profile: {
      id: -1,
      name: 'Coordinape',
      address: COORDINAPE_USER_ADDRESS,
      avatar: COORDINAPE_USER_AVATAR,
      skills: '',
},` as -1